Lombiq.ChartJs.Samples/Controllers/SampleController.cs

```diff
@@ -94,7 +94,7 @@ public async Task<IActionResult> Balance() =>
             {
                 // These options below are to simplify UI testing.
                 // You can find more detailed description here:
-                // https://www.chartjs.org/docs/2.9.4/configuration/
+                // https://www.chartjs.org/docs/4.4.7/configuration/
```

Comment:
How about using https://www.chartjs.org/docs/latest/configuration/ and similarly the `latest` version in the links above and below, so we don't have to keep updating these?

Reply:
I didn't intend to keep updating these, just wanted to pin the major version. In case they come up with a 5.0 where some of those links may not exist any more. But that's probably unlikely, so we can use latest.

Reply:
I'd expect that, yeah.